cli/src/pcluster/api/controllers/cluster_operations_controller.py

```diff
@@ -253,9 +256,32 @@ def describe_cluster(cluster_name, region=None):
         # This should not be treated as a failure cause head node might not be running in some cases
         LOGGER.info(e)
 
+    login_nodes = _get_login_nodes(cluster)
+    if login_nodes:
+        response.login_nodes = login_nodes
```

Comment:
[Nitpick] This is probably redundant.

Reply:
I prefer not to fix it for the time being.